examples/cisco/8000e/8000e-ixia.pb.txt

```diff
@@ -0,0 +1,130 @@
+name: "cisco-ixia"
+nodes: {
+    name: "vxr"
+    vendor: CISCO
+    model: "8201"
+    os: "ios-xr"
+    config: {
+        image: "ios-xr:latest" # please update the image
+        file: "r1.config"
+    }
+    constraints:{
+        key: "memory"
+        value: "20Gi"
```

Comment:
consider replacing the default constraint value: https://github.com/openconfig/kne/blob/bec917d6da9e8686fae0073190d0c64520362882/topo/node/cisco/cisco.go#L189

I would guess the majority of users would want to at least be able to enable gnmi and other services without increasing the default constraints

Reply:
done